postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -191,7 +195,7 @@ def shutdown_cb():
             check,
             rate_limit=1 / collection_interval,
             run_sync=is_affirmative(config.statement_samples_config.get('run_sync', False)),
-            enabled=is_affirmative(config.statement_samples_config.get('enabled', True)),
+            enabled=is_affirmative(config.statement_samples_config.get('enabled', True) or is_affirmative(config.statement_activity_config.get('enabled', True))), 
```

Comment:
maybe we can also add a new field to [this test](https://github.com/DataDog/integrations-core/blob/master/postgres/tests/test_statements.py#L923) where we disable statements to exercise this logic